### DISCUSSION TITLE
azure end point with api-key

### DISCUSSION THREAD

**Discussion by jomosi** [unit-mesh/auto-dev#118] [Q&A]
Could you please give me a example of the following instruction?

OpenAI Azure Config
...
"3. fill Custom OpenAI Host with your OpenAI API Endpoint. (with Key in URL)"

I have azure api end point (url type) and api-key for that end point, but how can I make end point with key in url ?

Thanks.

**Comment by phodal** [comment]
I don't have Azure endpoint, maybe try **end point with key in url**, or you can use other config.

**Reply by kapilboraste** [reply]
Any option for azure endpoint?

**Reply by phodal** [reply]
@kapilboraste try custom llm config, should be also works for Azure endpoint.
